develop/parachains/testing/benchmarking.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Benchmark Testing
+title: Benchmarking
 description: Learn how to use FRAME's benchmarking framework to benchmark your custom pallets and provide correct weights for on-chain computation and execution of extrinsics.
 ---
 
-# Benchmark Testing
+# Benchmarking
```

Comment:
The snippets in this file should be moved to the `develop/parachains/testing/benchmarking` dir and the paths be updated